### PR TITLE
Add: SEO - Google cards

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by firefinchdev** [comment]
Solves #9

**Comment by firefinchdev** [comment]
@abhishek97 
Done. Pls review.

File: utils/index.js [hunk 1]
```diff
@@ -2,10 +2,76 @@ const cheerio = require('cheerio')
...
+        "position": i+1,
+        "item": {
+          "@context": "http://schema.org",
+          "@type": "Course",
+          "name": course.title,
+          "description": course.subtitle,
+          "provider": {
+            "@type": "Organization",
+            "name": "Coding Blocks",
+            "sameAs": "https://codingblocks.com/classroom-courses/"
```
Comment (witty123) [inline utils/index.js]: online.codingblocks.com

File: utils/index.js [hunk 2]
```diff
@@ -2,10 +2,76 @@ const cheerio = require('cheerio')
...
+          "availability": "http://schema.org/InStock",
+          "seller": {
+            "@type": "Organization",
+            "name": "Coding Blocks"
+          }
+        }
+      }
+    }
+  }
+  return JSON.stringify(data, null, 4)
```
Comment (witty123) [inline utils/index.js]: remove indentation

File: utils/index.js [hunk 3]
```diff
@@ -14,6 +80,15 @@ const cookHTML = async (templateFileName, data) => {
       ${renderedTemplate}
     </noscript>
   `)
+  if (seoData !== undefined) {
+    console.log(cookSEO(seoData))
```
Comment (witty123) [inline utils/index.js]: remove console log

**Pull request merged**